impeller/geometry/matrix.h

```diff
@@ -305,6 +305,8 @@ struct Matrix {
 
   Scalar GetDeterminant() const;
 
+  bool IsInvertible() const { return GetDeterminant() != 0; }
```

Comment:
can you put these in the implementation?

Reply:
I'm not sure what you are asking?

Reply:
I forget the proper terminology :). Can you put the method bodies in the .cc file?

Reply:
These geometry files seem to prefer inline implementations unless the method is very complicated.